aws/src/integration/java/org/apache/iceberg/aws/lakeformation/TestLakeFormationAwsClientFactory.java

```diff
@@ -165,7 +165,8 @@ public void testLakeFormationEnabledGlueCatalog() throws Exception {
       glueCatalog.createNamespace(allowedNamespace);
     } catch (GlueException e) {
       LOG.error("fail to create Glue database", e);
-      Assert.fail("create namespace should succeed");
+
+      fail("create namespace should succeed");
```

Comment:
rather than using `fail` with a try-catch, it's better to have something like `assertThatNoException().isThrownBy(() -> glueCatalog.createNamespace(allowedNamespace))`

Reply:
Thanks. Will update this line.